add_gist.rb

```diff
@@ -48,7 +49,18 @@ def self.send_request(files, options = {})
       files: files
     }.to_json
 
-    http.request(request)
+    begin
+      http.request(request)
+    rescue StandardError => e
+      puts e.message
+      puts 'Would you like to resume (y/n)?'
```

Comment:
I would improve the message alerting on this feature

Reply:
Is this idea enough or should I use something like retriable? https://github.com/kamui/retriable

Reply:
now that you mention this I would like us to try it!